apps/analytics/analytics/sql/pv_pax.sql

```
@@ -0,0 +1,66 @@
+                                 ORDER BY x.ao_name, x.ao_org_id)
+                     FROM (SELECT DISTINCT ao_id AS ao_org_id, ao_name FROM observed
+                           WHERE user_id = u.user_id AND ao_id IS NOT NULL) x),
+                    []::STRUCT(ao_org_id INTEGER, ao_name VARCHAR)[]) AS aos
+    FROM user_base u LEFT JOIN pg.public.orgs hr ON hr.id = u.home_region_id
+)
+SELECT p.refreshed_at, r.user_id, r.f3_name, r.home_region_id, r.home_region_name,
+       r.avatar_url, r.status, r.start_date_override, r.regions, r.aos,
+       COALESCE((SELECT list(struct_pack(type_id := x.id, type_name := x.name) ORDER BY x.name, x.id)
+                 FROM (SELECT DISTINCT et.id, COALESCE(et.name, CAST(et.id AS VARCHAR)) AS name
+                       FROM observed o JOIN pg.public.event_instances_x_event_types xt ON xt.event_instance_id = o.event_id
+                       JOIN pg.public.event_types et ON et.id = xt.event_type_id WHERE o.user_id = r.user_id) x),
```

SQLFluff (4.3.0)

[error] The 'WHERE' keyword should always start a new line.

(LT14)